resources/reps-platform.make

```diff
@@ -17,6 +17,18 @@ projects[features_extra][version] = 1.0
 projects[features_roles_permissions][subdir] = "contrib"
 projects[features_roles_permissions][version] = 1.2
 
+; Add an option to delete all nodes when the feed source is empty.
+; https://www.drupal.org/project/feeds/issues/2333667
+; https://webgate.ec.europa.eu/CITnet/jira/browse/REPR-1719?focusedCommentId=3485574&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-3485574
+projects[feeds][subdir] = "contrib"
+projects[feeds][version] = 2.0-beta3
+projects[feeds][patch][] = "patches/feeds_beta3_delete_if_empty_source-2333667-15.patch"
+
+projects[mbp_sync][download][type] = git
+projects[mbp_sync][download][url] = https://github.com/ec-europa/mbp-sync-reference.git
+projects[mbp_sync][download][tag] = 1.0-rc1
+projects[mbp_sync][subdir] = custom
```

Comment:
custom or contrib?

Reply:
Custom. This module has been developed by Andras and now Victor Bourgade is taking the handover. Previously the module file were physically in the reps-platform. But since we are using a new version I'm taking the new version directly from github repo

Reply:
OK